kolibri/core/assets/src/vue/progress-bar/index.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+
+  <div>
+    <div class="visuallyhidden" id="progress-bar-label">{{ $tr('progressComplete') }}</div>
+    <div class="progress-bar-wrapper"
+      role="progress-bar"
+      aria-labelledby="progress-bar-label"
+      :aria-valuenow="progressPercent"
+      aria-valuemin="0"
+      aria-valuemax="100">
+      <div class="progress-bar-complete" v-bind:style="{ width: progressPercent + '%' }"></div>
+    </div>
+    <p class="progress-bar-text">{{ progressPercent + ' % ' + $tr('complete') }}</p>
```

Comment:
I'm just pondering if we should include `aria-live="polite"` here, but let's hold on it for now until we can test in action!